### DISCUSSION TITLE
Welcome to masonic Discussions!

### DISCUSSION THREAD

**Discussion by jaredLunde** [jaredLunde/masonic#139] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ghost** [comment]
Hi everyone.

**Comment by ktdickinson2** [comment]
@jaredLunde Your Masonic library is fantastic. I've spent the better part of today trying to get some other masonry libraries to work with React/Next.js and have run into a lot of challenges. Masonic worked like a dream straight out of the box and it's easy to understand the implementation. Thanks for the great work!
